.changeset/three-eyes-listen.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': patch
+---
+
+**This change updates a v5.0 breaking change when `experimental.directRenderScript` became the default script handling behavior**.
```

Comment:
Can you explain what the actual change is?

Reply:
This a copy-paste from the the changset we have in the compiler, so I don't have much context. Maybe @TheOtterlord can help here?

Reply:
This change was @MoustaphaDev (https://github.com/withastro/compiler/pull/1057). I don't have context here unfortunately.

Reply:
It's to enable conditional rendering of scripts, a feature `directRenderScript` was supposed to have in the first place. Previously the compiler implicitly made inline, scripts that were in expression blocks.

Reply:
```suggestion
Fixes an issue with the conditional rendering of scripts. 

**This change updates a v5.0 breaking change when `experimental.directRenderScript` became the default script handling behavior**.
```

Looks like this was the missing part